spec/openbd/resources/hanmoto_spec.rb

```diff
@@ -0,0 +1,129 @@
+    EOB
+  end
+
+  let(:sample_hanmoto_data2) do
+    # 9784863290006.json
+    <<-'EOB'
+{"dateshuppan":"2008-03","jyuhan":[{"date":"2016-06-23","comment":"とくに石垣が美しいことで有名な熊本城。本書はその姿をあえてモノクロームで表現しました。2016年4月の熊本地震で大きな被害をうけましたが、当時のうつくしく力強い姿をこれからに残すため、今回緊急増刷することにしました。","ctime":"2016-06-09 13:43:09","suri":3}],"author":[{"listseq":1,"dokujikubun":"編"}],"datemodified":"2011-03-19 11:43:10","zaiko":11}
+    EOB
+  end
+
+  let(:sample_hanmoto_data3) do
+    # 9784046215000.json
+    <<-'EOB'
+{"reviews":[{"han":"","appearance":"2010-03-14","post_user":"genkina","kubun_id":1,"source":"日本経済新聞","choyukan":"","source_id":23,"reviewer":"","link":""},{"han":"","appearance":"2010-03-21","post_user":"genkina","kubun_id":1,"source":"読売新聞","choyukan":"","source_id":20,"reviewer":"","link":""}],"datecreated":"2015-08-20 03:22:14","dateshuppan":"2010-02","datemodified":"2015-08-20 03:22:14"}
+    EOB
+  end
+
+  let(:hanmoto_item) do
+    json = JSON.parse(sample_hanmoto_data)
+    OpenBD::Resources::Hanmoto.new(json)
+  end
+
+  let(:hanmoto_item2) do
+    json = JSON.parse(sample_hanmoto_data2)
+    OpenBD::Resources::Hanmoto.new(json)
+  end
+
+  let(:hanmoto_item3) do
+    json = JSON.parse(sample_hanmoto_data3)
+    OpenBD::Resources::Hanmoto.new(json)
+  end
+
+  it "has a dateshuppan" do
+    expect(hanmoto_item.dateshuppan).to be == "2016-06"
+  end
+
+  it "has a ruishokyougousho" do
+    expect(hanmoto_item.ruishokyougousho).to be == "広井良典『人口減少社会という希望』朝日選書，加藤久和『世代間格差：人口減少社会を問いなおす 』ちくま新書など，人口問題に関する書籍。心理学的な観点に立った書籍は本邦初。"
+  end
+
+  it "has jyuhan" do
+    expect(hanmoto_item2.jyuhan[0].class).to be == OpenBD::Resources::Hanmoto::Jyuhan
+    expect(hanmoto_item2.jyuhan[0].date).to be == "2016-06-23"
+    expect(hanmoto_item2.jyuhan[0].comment).to be == "とくに石垣が美しいことで有名な熊本城。本書はその姿をあえてモノクロームで表現しました。2016年4月の熊本地震で大きな被害をうけましたが、当時のうつくしく力強い姿をこれからに残すため、今回緊急増刷することにしました。"
+    expect(hanmoto_item2.jyuhan[0].ctime).to be == "2016-06-09 13:43:09"
+    expect(hanmoto_item2.jyuhan[0].suri).to be == 3
+  end
+
+  it "has an obinaiyou" do
+    expect(hanmoto_item.obinaiyou).to be == "人口が減り始めた日本。\n私たちは命にどう関わるべきか？\n命についての問題―生殖補助医療，育児不安，母性，親子，介護，人生の終末―に直面し苦悩し，格闘する心を扱う「人口の心理学」の提案！\n心理学のみならず，人口学，社会学，生命倫理，日本近代史の第一線の論客が結集し，少子化，高齢化，人口減少に直面する日本社会のあり方を問う。"
```

Comment:
__[rubocop]__
* Metrics/LineLength: Line is too long. [210/200] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

<!-- Comment by [SideCI](https://sideci.com) -->